index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Hello Microverse">
+    <meta name="keywords" content="HTML, CSS, Linter">
+    <meta name="author" content="Oussedit">
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <header>
+            <a href="">
+                <div id="logo">My logo</div>
+            </a>
+            <nav>
+                <div><img src="./assets/ham.svg" alt="" > </div>
+            </nav>
+        </header>
+    </div>
+    <main>
+        <section id="sec1">
```

Comment:
- [ ] Good job applying the changes asked by the previous reviewer. but there are some problems in your design that was not there before. the heading and paragraph are overlapping and everything appears smaller.
I would suggest you apply changes to your design so that would be similar to the figma design.
your design now:
![sa](https://user-images.githubusercontent.com/96683843/211887956-495c4079-ff50-49e8-b747-efd225d3e530.jpg)

Figma design:
![sp1](https://user-images.githubusercontent.com/96683843/211888005-4e2d6ad4-ccc4-44f2-ac67-45fa79401b8f.jpg)